### ISSUE TITLE
Continuation of NoCheatPlus Discussion

### ISSUE THREAD

**Issue by ghost** [NoCheatPlus/Issues#587]
I'm opening this because the developer has left for so long, and none of the issues are being looked at.

Does anyone know how to reach asofold? Does anyone have an interest in trying to continue the plugin in his absence?

I've been waiting on this for so long now for 1.13, that I regret updating my server. I did not expect this.

If anyone has ideas on what to do with this situation, I'd appreciate any feedback.

Thanks!

**Comment by Janmm14** [comment]
Maybe try this: https://github.com/NoCheatPlus/Issues/issues/567#issuecomment-428488146

**Comment by ghost** [comment]
I have, everything works except the water at this point, and people have left my server as a result. :/

**Comment by MarkElf** [comment]
Configuration wise, using just NCP, I've yet to find an existing work around for water; additional plugins are another story. I did have someone reach out to me on spigot recently with interest to continue updating the plugin. I gave him a list of major issues that need attention, including the water movement setback. Still kind of waiting on that to see if anything happens though my hopes aren't set too high on that.

**Comment by CaptainObvious0** [comment]
I’ve started to work on a “band-aid” fix for NCP, while we can wait for it to be properly updated. Right now I’m just adjusting the values to not be as strict while in water which has worked for the most part, but I’m stuck on the issue of swimming through 1x1 spaces. NCP is a pretty complex plugin and it’s taking some time to understand it but it’s still pretty difficult. Once I get swimming figured out I want to move on to other major issues that can be fixed. 

Link to my NCP Fork: https://github.com/CaptainObvious0/NoCheatPlus

If you don’t want to compile it you can find the jar in the NCP config spigot thread. Let me know if you have any issues with it. This version can only be used on 1.13.

**Comment by ghost** [comment]
Nice I'm happy to hear someone's working on this. Players report that the water isn't perfect but it's way way better than before. :D

**Comment by CaptainObvious0** [comment]
Define "isn't perfect"

I just posted a current issues list on the spigot thread. If there are any issues not on that list please let me know.

**Comment by ghost** [comment]
@CaptainObvious0 I notice problems swimming to the surface, when slowly going upwards swimming it'll catch on the water surface. I think you found the other ones, unsure if you found this sone.

**Comment by MarkElf** [comment]
The guy that reached out to me to continue NCP through Spigot has gone silent for like a week now after I reached out twice. As I speculated, he probably wouldn't accomplish much.

Edit: apparently he doesn't want to release an unfinished project but I still doubt there will be any contribution from his end even after directing him to the fork.

**Comment by CaptainObvious0** [comment]
@SpacePuppeh Could you use the new version posted on the spigot thread and see if the issue continues? If it does, please record a video and post it to the issues tab of the respiratory.

**Comment by ghost** [comment]
Just confirming here @CaptainObvious0 it's fixed in the version ahead of the one I was using when I had the problem.
